GoogleSignIn/Tests/Unit/GIDGoogleUserTest.m

```diff
@@ -91,4 +110,64 @@ - (void)testLegacyCoding {
 }
 #endif // TARGET_OS_IOS || TARGET_OS_MACCATALYST
 
+- (void) testUpdateAuthState {
+  GIDGoogleUser *user = [self googleUser];
+  XCTAssertEqualObjects(user.accessToken.tokenString, kAccessToken);
+  [self assertDate:user.accessToken.expirationDate equalTime:_accessTokenExpireTime];
```

Comment:
I think you're trying to make your tests DRY here, but sometimes that can add some mental overhead to reading and debugging failing tests. Consider removing this and [making your tests DAMP](https://testing.googleblog.com/2019/12/testing-on-toilet-tests-too-dry-make.html).